src/app/core/components/generic-mod-list/generic-mod-list.component.html

```diff
@@ -23,9 +23,10 @@
       </div>
       <div class="mod-list-container">
         <strong class="mod-name one-line-clamp">{{ mod.name }}</strong>
+        <div class="mod-version">Last update: {{ getLastUpdateText(mod.lastUpdate) }}</div>
```

Comment:
I would suggest to add a smaller font size. Like font-size: small;
Because we dont have a lot of space left

Reply:
Set to small and also to take only 1 line max when window is too small, as to not push buttons down.